UPDATING.md

```diff
@@ -41,6 +41,13 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Changes to SQLSensor
+
+SQLSensor now consistent with python `bool()` function. It will resolve after receiving any value 
```

Comment:
This needs to explicitly mention that then `allow_null` parameter has been removed.